packages/core/src/toast/Toast.tsx

```diff
@@ -40,7 +45,12 @@ export const Toast = forwardRef<HTMLDivElement, ToastProps>(function Toast(
       ref={ref}
     >
       {status && (
-        <StatusIndicator status={status} className={withBaseName("icon")} />
+        <div
+          className={withBaseName("iconContainer")}
+          {...(icon && { "aria-hidden": true })}
```

Comment:
Are we sure this is what we want ? What if user passed in an icon with `aria-label`?